server/bed.js

```diff
@@ -52,12 +59,17 @@ bed.query.Motifs;
 /**
  * @param {!bed.query.Motifs} query
  * @param {string} bedPath
- * @return {!bed.MotifsResult}
+ * @return {!bed.MotifsResult|bed.Error}
```

Comment:
Typedef is by default non-nullable. There is no need for the exclaimation mark before bed.MotifsResult.